env.example

```diff
@@ -116,6 +116,10 @@ ETHERPAD_SKIN_VARIANTS="super-light-toolbar super-light-editor light-background
 # Enable guest access
 #ENABLE_GUESTS=1
 
+# Enable auth for Transcriber in Jigasi
+# This can only be enabled if the other auth options above are not used
+#JIGASI_TRANSCRIBER_AUTHENTICATION=1
```

Comment:
This is very confusing. The transcriber should be authenticated always, like a recorder. Can this be dropped?

Reply:
I guess that eventually, this can be dropped and the authentication of the transcriber can be made mandatory. I don't know whether this would be a "breaking change" though? Maybe it was never meant that the transcriber works without authentication, but I can't tell. Please see also my generic comment below.